installer.cmd

```diff
@@ -0,0 +1,257 @@
+::@Kimbyeongjang - (김병장#7165)
+@echo off
+title A32NX Builder
+
+>nul 2>&1 "%SYSTEMROOT%\system32\cacls.exe" "%SYSTEMROOT%\system32\config\system"
+if '%errorlevel%' NEQ '0' (
+    echo Request Administrator Privilege ...
+    goto UACPrompt
+) else ( goto gotAdmin )
+
+:UACPrompt
+    echo Set UAC = CreateObject^("Shell.Application"^) > "%temp%\getadmin.vbs"
+    set params = %*:"=""
+    echo UAC.ShellExecute "cmd.exe", "/c %~s0 %params%", "", "runas", 1 >> "%temp%\getadmin.vbs"
+    "%temp%\getadmin.vbs"
+    rem del "%temp%\getadmin.vbs"
+    exit /B
+
+:gotAdmin
+pushd "%CD%"
+    CD /D "%~dp0"
+	
+:init
+docker version
+if "%ERRORLEVEL%" == "0" ( 
+     echo. Docker check passed...
+) else ( 
+     goto dockerERROR
+)
+
+git version
+if "%ERRORLEVEL%" == "0" ( 
+     echo. Git check passed...
+) else ( 
+     goto gitERROR
+)
+
+goto main
+
+:dockerERROR
+cls
+echo.
+echo ===================================================================
+echo               You should install Docker for build
+echo                 Do you want to install Docker?
+echo ===================================================================
+echo.
+echo
+set /p docker= Y or N : 
+if "%docker%"=="y" (

```

Comment:
You need the `/i` here to handle uppercase (e.g. `Y`) too. Learn more here: https://ss64.com/nt/if.html

This also applies further down.

```suggestion
if /i "%docker%"=="y" (
```